### DISCUSSION TITLE
After the version is updated, the PDF preview is blurry

### DISCUSSION THREAD

**Discussion by 651899918** [James-Yu/LaTeX-Workshop#4492] [Q&A]
After the version is updated, the fonts in the preview window of the generated PDF are rather blurry. They only look normal when enlarged.
![PixPin_2024-12-27_16-30-17](https://github.com/user-attachments/assets/ba83349a-70e8-47dd-8f02-20cc2a30aac5)

**Comment by npulidomateo** [comment]
I'm observing a similar effect. Version 10.10.0

**Comment by zzLearning** [comment]
I'm currently using version 10.11.2, but the fonts are still blurry. Are there any solutions available?

**Reply by maniok098** [reply]
Downgrading to `10.10.2` fixed this issue for me. It seems to be related to the update of `PDF.js` since version `10.11.0`.

**Comment by jpmvferreira** [comment]
I have the same problem. Text of similar size in VS Code looks clear but in the PDF viewer not so much.
